src/daemons/MetaDaemon.cpp

```diff
@@ -170,7 +171,8 @@ int main(int argc, char *argv[]) {
     }
 
     google::SetVersionString(nebula::versionString());
-    folly::init(&argc, &argv, true);
```

Comment:
why remove the init method.

Reply:
Because folly::init is almost same with init `gflags + glog + singleton`. You mean we init gflags twice?